### DISCUSSION TITLE
Multi-column great expectations

### DISCUSSION THREAD

**Discussion by flaneur-ml** [great-expectations/great_expectations#5240] [Support]
**Is your feature request related to a problem? Please describe.**
I am struggling with including the multi-column great-expectations for my current projects. For instance, given a list of columns I want to check if each next column values are higher than the values in previous one. Is the any way it can be implemented through custom expectations? In the documentation only the templates for one-column expectations can be found. 

**Describe the solution you'd like**
Ideally, there could be the template for the expectations which take the list of columns as inputs and some values as arguments which can be overridden depending on the functionality required. 

**Describe alternatives you've considered**
I considered to implement a function which would take the list of columns and produce one column with 0 where the values are rising and 1 otherwise and then use the `expect_column_values_to_not_be_in_set`. However, this solution seems bulky and requires new function to be added to the project.

**Comment by AFineDayFor** [comment]
Howdy @youth-quaker, thanks for reaching out and being a part of the community 🎉

I'm moving this over to discussions so it can be referenced as a more permanent resource due to the nature of the question.

Using [evaluation_parameter](https://docs.greatexpectations.io/docs/terms/evaluation_parameter)s may help you get this over the line.

**Reply by flaneur-ml** [reply]
Thanks for your reply. The evaluation parameters are used when validation against the previous runs is required. I was rather referring to the multi-column great_expectations inside the expectation suite.
